Makefile

```diff
@@ -107,23 +107,19 @@ update-vendor:
 	hack/update-vendor.sh
 
 .PHONY: unit-test
-unit-test: autogen
+unit-test: update-vendor
 	hack/unit-test.sh
 
-.PHONY: install-etcd
-install-etcd:
-	hack/install-etcd.sh
-
-.PHONY: autogen
-autogen: update-vendor
-	hack/update-generated-openapi.sh
```

Comment:
@yibozhuang Removing this script is causing the following error for me when I try to run intg tests:

```
+++ [0308 22:31:18] Configuring envtest
+++ [0308 22:31:18] Running integration test cases
# k8s.io/kubernetes/cmd/kube-apiserver/app
vendor/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:401:80: undefined: "k8s.io/kubernetes/pkg/generated/openapi".GetOpenAPIDefinitions
```

Earlier running `make autogen` was used to fix this. I tried running `update-vendor` but it is not helping.

CC: @Huang-Wei

Reply:
See https://github.com/kubernetes-sigs/scheduler-plugins/pull/344#discussion_r813126976

Basically if your integration test is importing `k8s.io/kubernetes/test/integration/util` then you will hit this problem.
So I suggest taking a look at the `utils.go` at https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/test/integration/utils.go and update your test to remove any dependency on the k/k `test/integration/util` package.

Reply:
Neat!

Reply:
@zorro786 now we get rid of hack/update-generated-openapi.sh :)